src/services/services.ts

```diff
@@ -2537,9 +2537,9 @@ namespace ts {
             }
         }
 
-        function getRenameInfo(fileName: string, position: number, preferences: UserPreferences): RenameInfo {
+        function getRenameInfo(fileName: string, position: number, preferences: UserPreferences | RenameInfoOptions | undefined): RenameInfo {
             synchronizeHostData();
-            return Rename.getRenameInfo(program, getValidSourceFile(fileName), position, preferences);
+            return Rename.getRenameInfo(program, getValidSourceFile(fileName), position, preferences || {});

```

Comment:
I thought you said we needed to handle old-style calls specially (probably by skipping the new `node_modules` checks)?

Reply:
We clarified offline that the scenario I had in mind doesn't really exist for API consumers.